content/rs/administering/cluster-operations/maintenance-mode.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: Maintenance Mode for Cluster Nodes
+description: Prepare node for maintenance, and restore it to previous state
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+When you need to do hardware or operating system maintenance on an RS server,
+it is important that you move all of the shards on that server to another node to avoid the risk of data loss.
+You can use maintenance mode to handle this process simply and efficiently.
+
+## Turning Maintenance Mode ON
+
+When you turn maintenance mode on, RS:
+
+1. Checks whether shutdown of the node causes quorum loss. If so, maintenance mode is not turned on.
```

Comment:
Just as a reminder- all cluster nodes can be moved to maintenance mode with no problem. One cannot turn on maintenance mode only in case few nodes are already down. Is that correct?

Reply:
@RoeyPrat ^^^

Reply:
yes, maintenance mode can be turned on on all nodes - but only if there are no databases.

one cannot turn on maintenance mode in the following cases:
1. if shutdown of this node can cause quorum loss (all other things being the same)
2. if shards could not be migrated